sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -1,17 +1,43 @@
 """A script that queries AWS API to get instance types and pricing information.
-
 This script takes about 1 minute to finish.
 """
 import datetime
-from typing import Tuple
+from typing import Tuple, Union
 
 import numpy as np
 import pandas as pd
 import ray
 
 from sky.adaptors import aws
 
-REGIONS = ['us-east-1', 'us-east-2', 'us-west-1', 'us-west-2']
+ALL_REGIONS = [
+    'us-east-1',
+    'us-east-2',
+    'us-west-1',
+    'us-west-2',
+    'ca-central-1',
+    'eu-central-1',
+    'eu-west-1',
+    'eu-west-2',
+    'eu-south-1',
+    'eu-west-3',
+    'eu-north-1',
+    'me-south-1',
+    # 'me-central-1', # failed for no credential
+    'af-south-1',
+    'ap-east-1',
+    'ap-southeast-3',
+    # 'ap-south-1', # failed for no credential
+    'ap-northeast-3',
+    'ap-northeast-2',
+    'ap-southeast-1',
+    'ap-southeast-2',
+    'ap-northeast-1',
```

Comment:
nit: Could you alphabetically sort the items, if there's no particular reason for this order?

Reply:
Sounds good. Thanks for the explanation.